app/src/screens/Splash.tsx

```diff
@@ -24,77 +22,11 @@ import TipCarousel from '../components/TipCarousel'
 import useInitializeBCAgent from '../hooks/initialize-agent'
 import { BCState } from '../store'
 
-import { TermsVersion } from './Terms'
-
-const OnboardingVersion = 2
```

Comment:
So `OnboardingVersion` just isn't needed anymore? That's awesome

Reply:
No, I can't think of any reason to keep it. We do use TermsVersion.